app_connections/controllers/review.controller.js

```diff
@@ -1,5 +1,16 @@
 const { response } = require("../app");
-const { fetchReview, editReview } = require("../models/review.model");
+const {
+  fetchReview,
+  editReview,
+  fetchAllReviews,
+} = require("../models/review.model");
+
+exports.getAllReviews = (req, res) => {
+  fetchAllReviews().then((reviews) => {
+    console.log(reviews);
+    res.status(200).send({ reviews });
+  });
```

Comment:
dont forget to add a catch block to all of your promise chains, this is the difference between your app crashing all together and sending back a 500